source/index.txt

```diff
@@ -17,6 +17,7 @@ MongoDB PHP Library
    /aggregation
    /indexes
    /security
+   /read-write-pref
```

Comment:
S: I think that this section might be better suited to be under the read/write drawers

Reply:
I wasn't sure whether to put it under read or write - I guess read if I had to choose?

Reply:
I dont think you need to put it in a drawer but just in the list under read and write